microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -32,9 +32,10 @@ class URIHandler:
     """
     __metaclass__ = ABCMeta
 
-    def __init__(self, graph, nack_timeout=1):
+    def __init__(self, graph, nack_timeout=1, resource_nack_timeout=1):
```

Comment:
Suggestion: rename both timeout variables for clarity and use an `@property` for the legacy `nack_timeout` usage compatibility (until it is fixed)

Reply:
Sure.